manifests/init.pp

```diff
@@ -270,6 +274,7 @@
   String $repo_yum_gpgkey                                       = $rundeck::params::repo_yum_gpgkey,
   Stdlib::HTTPUrl $repo_apt_source                              = $rundeck::params::repo_apt_source,
   String $repo_apt_key_id                                       = $rundeck::params::repo_apt_key_id,
+  String $repo_apt_gpgkey                                       = $rundeck::params::repo_apt_gpgkey,
```

Comment:
```suggestion
  String[1] $repo_apt_gpgkey                                    = $rundeck::params::repo_apt_gpgkey,
```

Reply:
The `apt::key` seems to support http, https, ftp and local path:

https://github.com/puppetlabs/puppetlabs-apt/blob/df5a5f4d2441258f93207be6edd814549c36ce19/manifests/key.pp#L42

I am not sure a user will ever want to tweak all these repository settings however, so maybe using a more conservative type here might really not be an issue :shrug:

Reply:
otherwise we can also take `Pattern[/\Ahttps?:\/\//, /\Aftp:\/\//, /\A\/\w+/]`